### PR TITLE
adds fallback image for when someone doesnt have a portrait

### PR DESCRIPTION
**Pull request by JonathanOppenheimer**
i will be 30 years old and andrew will still be cleaning up my messes for me <3

### CODE REVIEW

**Comment by AndrewLester** [comment]
Remove console log for success then it's good. It'll be spam.

**Comment by JonathanOppenheimer** [comment]
image
i can confirm it works locally with the console logs delete

**Comment by AndrewLester** [comment]
What key is that sir. Looks good

**Comment by JonathanOppenheimer** [comment]
me rn
image

**Review by AndrewLester — APPROVED** [review]

File: scripts.js [hunk 1]
```diff
@@ -229,6 +229,15 @@ function details(institution) {
             studentName = document.createElement('p'),
             studentMajor = document.createElement('p');
         studentPhoto.src = 'portraits/' + currentYear + '/' + student.name + '.jpg';
+        imageExists(studentPhoto.src, function(exists) {
+            if(exists == true)
```
Comment (ErikBoesen) [inline scripts.js:233]: ‼️

Comment (ErikBoesen) [reply scripts.js:233]: You can probably even make this whole thing a ternary statement. At the very least, replace with `if (exists) {`

**Pull request merged**